build-system/tasks/visual-diff/index.js

```diff
@@ -44,6 +44,8 @@ const WEBSERVER_TIMEOUT_RETRIES = 10;
 const NAVIGATE_TIMEOUT_MS = 3000;
 const MAX_PARALLEL_TABS = 10;
 const WAIT_FOR_TABS_MS = 1000;
+const AMP_RUNTIME_TARGET_FILES = [
+  'dist/amp.js', 'dist.3p/current/integration.js'];
```

Comment:
This list of files has grown considerably.

https://github.com/ampproject/amphtml/blob/fe4d16908df061955a5a5aa75f121794a61647da/gulpfile.js#L76-L81

Reply:
I tried adding `dist/amp-esm.js` and it caused a crash, because this file wasn't built. I think visual diff tests only use `dist/amp.js` and `dist.3p/current/integration.js`, so there's no need to add them all

Reply:
Yeah, you'll need to read the logic lower down in `gulpfile.js`.